### ISSUE TITLE
Date Picker: Accessibility Issues

### ISSUE THREAD

**Issue by coliff** [trimble-oss/modus-web-components#2283]
- **This button should be `disabled`**
![image](https://github.com/trimble-oss/modus-web-components/assets/1212885/571ea2f0-e4e0-4a77-b13c-a4584001904b)

- There are multiple `aria-current="date"` - there should only be 1.

![image](https://github.com/trimble-oss/modus-web-components/assets/1212885/9f4ef0fa-8555-4458-a39f-4b33f38aca13)

- **Remove invalid `role="title"` and unnecessary `aria-label="ModusCalendar title"` from Month View Popup** (FIXED)
![image](https://github.com/trimble-oss/modus-web-components/assets/1212885/710b3020-371e-4fa0-973c-937015dceb7b)
